core/src/test/java/org/elasticsearch/search/searchafter/SearchAfterBuilderTests.java

```diff
@@ -214,14 +215,15 @@ public void testFromXContent() throws Exception {
         for (int runs = 0; runs < 20; runs++) {
             SearchAfterBuilder searchAfterBuilder = randomJsonSearchFromBuilder();
             XContentBuilder builder = XContentFactory.contentBuilder(randomFrom(XContentType.values()));
-            if (randomBoolean()) {
-                builder.prettyPrint();
-            }
             builder.startObject();
             searchAfterBuilder.innerToXContent(builder);
             builder.endObject();
-            XContentParser parser = XContentHelper.createParser(builder.bytes());
-            QueryParseContext context = new QueryParseContext(indicesQueriesRegistry, parser, ParseFieldMatcher.STRICT);
+            XContentBuilder shuffled = shuffleXContent(builder, Collections.emptySet());
+            if (randomBoolean()) {
+                shuffled.prettyPrint();
```

Comment:
I'm pretty sure you have to call `prettyPrint` early if you are going to use it? This is too late?